src/Http/Faking/MockClient.php

```diff
@@ -159,7 +159,7 @@ public function guessNextResponse(PendingRequest $pendingRequest): MockResponse|
         }
 
         if (empty($this->sequenceResponses)) {
-            throw new NoMockResponseFoundException;
+            throw new NoMockResponseFoundException($request);
```

Comment:
I'm just wondering - would you want the domain + path in the debug log? If you pass the PendingRequest through here, you can do `$pendingRequest->getUrl()` I believe which should return something like: `https://some-api.com/api/path`.

Reply:
I would say that it depends on how Saloon::fake([]) will work in v3, thinking regarding my feedback https://github.com/saloonphp/saloon/discussions/219#discussioncomment-6616142. If Saloon::fake() will continue to only expect `/api/path` then I would prefer to get that. But if it will work to send full URL to Saloon::fake() in the future then yeah full URL would be more nice.